### DISCUSSION TITLE
Welcome to FsProjectsAdmin Discussions!

### DISCUSSION THREAD

**Discussion by dsyme** [fsprojects/FsProjectsAdmin#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nojaf** [comment]
Hi all,

My name is Florian and I maintain the Fantomas project (https://github.com/fsprojects/fantomas and https://github.com/fsprojects/fantomas-tools).
Saying that out loud, makes it sound like an AA meeting. Chuckles 😅.
If you have any questions on Fantomas, feel free to reach to our [gitter](https://gitter.im/fsprojects/fantomas).

Anyway, I hope to get to know some more maintainers of the `fsprojects` group. I think we can benefit from this and gain insights into how other people are managing their projects.

Happy to be here!

Cheers

**Comment by dsyme** [comment]
@sergey-tihon We should probably start an "fsprojects maintainers" team that includes everyone (and add it to policies), to allow conversation/discussion.

**Reply by sergey-tihon** [reply]
Done https://github.com/fsprojects/fsprojects-admin/discussions/36
